From 297ca10bb4356c0e256ad502277c9bbd3b14cbcd Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Tue, 17 Jul 2012 14:17:30 +0200
Subject: [PATCH] Don't use rgb.txt from pstoimg 2 of Debian series 7  This
 patch prevents pstoimg from using rgb.txt which can make
 latex2html calls  fail. Author: Atsuhito KOHDA
 <kohda@pm.tokushima-u.ac.jp>

---
 pstoimg.pin |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pstoimg.pin b/pstoimg.pin
index a9f0777..876f6cd 100755
--- a/pstoimg.pin
+++ b/pstoimg.pin
@@ -1508,7 +1508,8 @@ sub crop_scale_etc {
       $pnmtoimg .= ' -interlace';
     }
     if($TRANSPARENT) {
-      $trans_color = $TRANSPARENT_COLOR||'gray85';
+     $trans_color = $TRANSPARENT_COLOR||'#d9d9d9'; # don't lookup rgb.txt
+     #$trans_color = $TRANSPARENT_COLOR||'gray85';
       $pnmtoimg .= ' -trans ' . L2hos->quote($trans_color);
     }
   }
-- 
1.7.10.3

